app/src/services/dictionary-service.js

```diff
@@ -63,6 +63,7 @@ angular.module('otServices')
             // N
             NA: 'N/A',
             NO_DATA: 'No data',
+            NFERX :'Literature - NFERX',
```

Comment:
same here... we are moving this to a config json, but you will need to keep it here until it is done